pkg/ccl/backupccl/backup_job.go

```diff
@@ -406,7 +406,8 @@ func (b *backupResumer) Resume(ctx context.Context, execCtx interface{}) error {
 
 	var backupManifest *backuppb.BackupManifest
 
-	// If planning didn't resolve the external destination, then we need to now.
+	// If the backup job has already resolved the destination in a previous
+	// resumption, we can skip this step.
 	if details.URI == "" {
```

Comment:
This might be a good block to break out into its own function? In a followup diff or TODO would be fine with me too.

Reply:
actually i'm going to avoid eating another CI cycle and add the TODO in https://github.com/cockroachdb/cockroach/pull/82381 if thats cool